modules/vorbis/audio_stream_ogg_vorbis.cpp

```diff
@@ -546,6 +550,10 @@ void AudioStreamOggVorbis::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "loop_offset"), "set_loop_offset", "get_loop_offset");
 }
 
+Ref<AudioStreamOggVorbis> AudioStreamOggVorbis::from_filesystem(const String &p_path) {
+	return ResourceImporterOggVorbis::import_ogg_vorbis(p_path);
```

Comment:
This is only compiled in editor builds if I remember right. This was an issue brought up in the previous PR that I had overlooked. I expect export templates to not build with this call.

Reply:
<img width="336" alt="image" src="https://github.com/godotengine/godot/assets/32321/397e1d48-349a-4bd4-a74c-01f966df2b64">

It didn't seem to happen in this pr.